src/runtime_src/tools/scripts/xrtdeps.sh

```diff
@@ -276,15 +289,18 @@ prep_rhel()
     echo "Enabling EPEL repository..."
     rpm -q --quiet epel-release
     if [ $? != 0 ]; then
-	yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+	if [ $MAJOR -lt "8" ]; then
+	    yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+	else
+	    yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
+	fi
```

Comment:
Let's move this in prep_rhel7 and prep_rhel8.  Also move cmake3 install after prep_rhel8 and rhel7 ?

Reply:
@stsoe made changes as suugested, please review